### PR TITLE
Milestone 1-6: Bootstrap

### PR DESCRIPTION
**Pull request by jebitok-dev**
### Newsweek Replica Milestones:

In these milestones we:
- Set up the repository and prepared assets for our website.
- Analyzed different parts of the website and created the skeleton
- Applied styles and positioning using bootstrap classes 
- Used Bootstrap classes to create different viewports of the website.

### CODE REVIEW

**Comment by mimipeshy** [comment]
## STATUS:  CHANGES REQUIRED 🔴 ♻️ 
Hello team @jebitok-dev  and @Leandro-Barretoo  👋 👋 , good job on the project so ar, it looks good but let's make it better 😄

#### Large view
- [ ]  The navigation bar should resemble the original one [link to page](https://www.newsweek.com/)
Original
o


Clone
c

#### Medium view

- [ ]  The navigation bar should resemble the original one [link to page](https://www.newsweek.com/)

Original
o

Clone
c

- [ ]  Ensure the whole page looks exactly like the original one [link to page](https://www.newsweek.com/)
Original


Clone
or


- [ ]  Give margins between the articles in the world, business, tech & science, sports, and US section [link to page](https://www.newsweek.com/)

Original

or

Clone
c

- [ ] One image is missing in the subscribe section. Kindly add it and align as it looks in the original page. [link to page](https://www.newsweek.com/)
Original
sub

Clone
c

#### Mobile view

- [ ]  Top story should be the first article in this view [link to page](https://www.newsweek.com/)

- [ ] Navigation bar should resemble  the original one [link to page](https://www.newsweek.com/)

Original
or

Clone
c

- [ ] Ensure the in magazine section looks like the original one  [link to page](https://www.newsweek.com/)

Original
image


Clone
image

- [ ]  Editor's pick should look like the original one

Original
image


Clone
image


- [ ] Take a minute to ensure the whole page looks like the original one  [link to page](https://www.newsweek.com/)

- [ ] Remove the commented code in your HTML file


That's all. 😄 , kindy submit another review when you are done

Regards
Peris Ndanu
Feel free to reach out on slack if you have any questions

File: stylesheets/app.css [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+  width: 300px;
+  height: auto;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.fa-bars {
+  display: none;
+}
```
Comment (andresporras3423) [inline stylesheets/app.css]: Same style definition of the selector on line 67, group selectors with the same style definition

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @wrakc** 
Good job with your first submission for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [ ] Your footer only have 4 of 5 social media icons from the original, add the Instagram icon
- [ ] The page should use two break points (like the original) and not only one

| Yours | Original |
|:---:|:---:|
|image | image |

- [ ] Group selectors with the same style definition
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] The github actions linter configuration is not correct, the linters are not running, please fix this.
image


- [ ] The branch has some issues please fix them
image

- [ ] The website is non-responsive. See notes below:

> * [x]  Your footer only have 4 of 5 social media icons from the original, add the Instagram icon 
- [ ]  The page should use two breakpoints (like the original) and not only one. **The project has two break points to give you a hint they are 768px and 425px** 

These breakpoints go like this:
Desktop above 769px;
Tablet view between 768px to 426px;
Mobile view below 425px;

This means that the project should have 3 views, Desktop, Tablet, and Mobile.

Right now the project has a mixture of Tablet and Mobile and is making the Desktop view.

Example:
#### Original

| Desktop | Tablet | Mobile |
| - | - | - |
| image | image | image |

#### Clone

| Desktop | Tablet | Mobile |
| - | - | - |
| image | image | image |

- [ ]  Group selectors with the same style definition. **The stylesheet still has repeated CSS rules, keep the code DRY and join them together**

 Example:
```css
# Bad CSS
.element__one {
  color: red;
}

.element__two {
  color: red;
}

/* Good CSS option 1 */

.element__one,
.element__two {
  color: red;
}

/* Good CSS option 2 */

.color-red {
  color: red;
}
```

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hello team @jebitok-dev and @Leandro-Barretoo  👋 👋 
✔️  Good job on working on the TSE feedback you were given
✔️ Your project is now taking shape, its almost resembling the original page
⚠️ Don't give up yet because of all the changes you are being told to make, we want you to be the best developers

Let's make a few more changes before we mark the project as `complete`

**Note**: use bootstrap classes to achieve the changes, using CSS styles is okay but will take you more time to make all the changes that could easily solved with a simple bootstrap class

### MANDATORY 🔴 

- [ ]  The whole project is missing the tablet breakpoint which is 768px. Kindy fix that . `col-md` is used in this breakpoint. Since this project is bootstrap-based, you should make use of bootstrap classes to order and align elements instead of CSS.

- [ ] In the debate section you are missing the `vs` icon just between the two articles

Original | Clone
------------ | -------------
image |image


Notice the difference ❓ ❓ 

- [ ] On tablet view, the `IN THE MAGAZINE ` section should be aligned to 4 articles per row. Use bootstrap classes to achieve this rather than writing CSS styles.

- [ ] On a tablet view, the `Editor's Picks` section should be aligned with two articles per row. Use bootstrap classes to achieve this.

- [ ] On the tablet view, this section is missing. Add it
image


- [ ] On the desktop view, there are issues with alignment in the `IN THE MAGAZINE` section. Kindly fix that

Original | Clone
------------ | -------------
image |image

Notice the difference ❓ ❓ 

- [ ] On desktop view the `editors section` also has alignment issues

Original | Clone
------------ | -------------
image |image

Notice the difference ❓ ❓ 

- [ ] On the desktop view, this section is missin too. Add it
image

- [ ] This section `US, WOLRD, POLITICS` etc too needs margins between each article
image

- [ ] The buttons on the `subscribe` and `newsletter` section are flowing out of the container. Kindly fix that
image


That's all 😄 kindly submit another review when you are done.

Regards
Peris Ndanu
Feel free to slack me anytime if you have any questions.

**Comment by mimipeshy** [comment]
## STATUS :  CHANGES REQUESTED 🔴 ♻️ 

Hey team 👋 👋 it's me again 😄 
✔️ Great job on working on the requirements above, your project is looking really good.

However, before we approve it lets fix one more thing

- [ ] On mobile and tablet view, the contents below editor's section and above the subscribe section, should be hidden.

That's all. 😄 , kindy submit another review when you are done

Regards
Peris Ndanu
Feel free to reach out on slack if you have any questions

**Comment by mimipeshy** [comment]
## STATUS : APPROVED 🥳 🍾 

Hey team @jebitok-dev and @Leandro-Barretoo, I mark this project as approved. You have met all requirements needed. Go ahead and merge. 

All the best in future projects. happy coding 💻 


Regards
Peris Ndanu

**Pull request merged**